freezer/freezer_url.go

```diff
@@ -31,19 +32,35 @@ func newFreezerSink(u *url.URL) (substrate.AsyncMessageSink, error) {
 		return nil, fmt.Errorf("unknown compression type : %s", cts)
 	}
 
+	var streamstore straw.StreamStore
 	switch u.Scheme {
 	case "freezer+dir":
-		conf := AsyncMessageSinkConfig{
-			streamstore: &straw.OsStreamStore{},
-			fconfig: freezer.MessageSinkConfig{
-				Path:            u.Path,
-				CompressionType: ct,
-			},
+		streamstore = &straw.OsStreamStore{}
+	case "freezer+s3":
+		var enc straw.S3Option
+		if q.Get("sse") == "aes256" {
```

Comment:
It would be nice if you could add this option to the freezer source as well and document this option in the [doc.go](here https://github.com/uw-labs/substrate/blob/master/freezer/doc.go) file.

Reply:
i don't think it makes a difference for the source?

Reply:
@SpeedyCoder do you mean add a config option for "sse" instead of just hard coding `aes256` ?

Reply:
ah cool, wasn't sure if it makes a difference for the source 👍

Reply:
Might be worth returning an error if something other than `aes256` is provided as an `sse` option